packages/babel-preset-env/package.json

```diff
@@ -22,7 +22,7 @@
     "@babel/helper-validator-option": "workspace:^",
     "@babel/plugin-bugfix-safari-id-destructuring-collision-in-function-expression": "workspace:^",
     "@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining": "workspace:^",
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.0",
+    "@babel/plugin-proposal-private-property-in-object": "condition:BABEL_8_BREAKING ? : 7.21.0-placeholder-for-preset-env.2",
```

Comment:
Could this ternary require two values?

Reply:
This ternary is just to make sure that the dependency will be automatically removed in Babel 8, in the published package it's being resolved to `7.21.0-placeholder-for-preset-env.2`